packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -78,15 +81,16 @@ export interface ComboBoxNextProps extends Omit<ComponentPropsWithoutRef<"input"
   /**
    * Item renderer function.
    */
-  itemRenderer?: (
+  itemRenderer: (
```

Comment:
Does this need to be mandatory prop seeing as you assign it the default renderer on 117 ? 
Same for Item Filter